cmd/ethereum_rust/ethereum_rust.rs

```diff
@@ -80,11 +81,15 @@ async fn main() {
     let tcp_socket_addr =
         parse_socket_addr(tcp_addr, tcp_port).expect("Failed to parse addr and port");
 
-    let mut store = match matches.get_one::<String>("datadir") {
-        Some(data_dir) if !data_dir.is_empty() => Store::new(data_dir, EngineType::Libmdbx),
-        _ => Store::new("storage.db", EngineType::InMemory),
-    }
-    .expect("Failed to create Store");
+    let project_dir = ProjectDirs::from("com", "lambdaclass", "ethereum_rust")
```

Comment:
java?

Reply:
The one true directory tree.

Reply:
I think it is valid to leave the two first arguments empty. I'm not 100% sure.

Reply:
I would just do `ethereum_rust ` tbh

Reply:
Done

Reply:
Now it should be like this:
- Mac: `~/Library/Application Support/ethereum_rust`
- Linux: `~/.config/ethereum_rust`

Reply:
nice!